assets/scss/01-atoms/_input-number.scss

```diff
@@ -55,16 +55,16 @@
     transform: rotate(180deg);
   }
 }
-
-//theme
+
+//theme

```

Comment:
Not a big deal, or something that is needed in this PR, but as I have went through the theme files, and this `//theme` section still exists, I have been trying to integrate them and clean up the files.